creator-node/src/snapbackSM/snapbackSM.js

```diff
@@ -92,13 +103,21 @@ class SnapbackSM {
       return
     }
 
+    // Setup the mapping of Content Node endpoint to service provider id. Used in reconfig
```

Comment:
we'll probably need to refresh this data set on some interval right? will get stale if only retrieved once on node startup

Reply:
discussed offline - update each modulo slice